compose/material3/adaptive/adaptive-layout/build.gradle

```diff
@@ -139,6 +139,40 @@ kotlin {
     }
 }
```

Comment:
Let's also add `StringsTest`, similar to material module

Reply:
I meant the test that verifies that all fields have translation, we need to have it on desktop, and use reflection for that.

Reply:
done